docs/source/introduction.rst

```diff
@@ -91,8 +91,7 @@ build it from you will need:
 * Preqrequisites of `mgclient`_:
 
   * CMake 3.8 or newer
-  * OpenSSL 1.0.2 or newer
-  * OpenSSL 1.0.2 or newer header files
+  * OpenSSL 1.0.2 or newer (but not OpenSSL 3.0) and its header files
```

Comment:
Will this version work for macos11? If that's the case, then why the line above is not
`$ brew install openssl@1.0` instead of `$ brew install openssl@1.1`

Reply:
That's true, we don't consider that case in `setup.py` when we are dealing with `possible_openssl_root_dirs`. I will check that how it can be done properly.

Reply:
Currently only 1.1 and 3.0 versions are available to install by brew, so technically you can install pymgclient with OpenSSL 1.0.2, but you have to build it by yourself if you want to do that. Therefore I won't spent time on describing this.